Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -112,6 +116,9 @@ class BrowserViewController: UIViewController, BrowserToolbarDelegate, TabManage
                 if let progress = change[NSKeyValueChangeNewKey] as Float? {
                     toolbar.updateProgressBar(progress)
                 }
+
+                toolbar.updateBackStatus(object.canGoBack)
+                toolbar.updateFowardStatus(object.canGoForward)
```

Comment:
I don't think we want to update the back/forward state every time the load progress amount changes. Perhaps UIWebViewDelegate#webViewDidStartLoad would be a suitable place to handle this.